sdk/resources/arm-resources/api-extractor.json

```
@@ -0,0 +1,18 @@
+{
+  "$schema": "https://developer.microsoft.com/json-schemas/api-extractor/v7/api-extractor.schema.json",
+  "mainEntryPointFilePath": "./esm/index.d.ts",
+  "docModel": { "enabled": true },
+  "apiReport": { "enabled": true, "reportFolder": "./review" },
+  "dtsRollup": {
+    "enabled": true,
+    "untrimmedFilePath": "",
+    "publicTrimmedFilePath": "./esm/index.d.ts"
```

@sarangan12 also please implement this change as well to follow what Track 2 data-plane packages do.
```suggestion
    "publicTrimmedFilePath": "./types/arm-resources.d.ts"
```

---

@deyaaeldeen Could you please explain what is the impact of not changing it for this release? (We can change it for the next one). Will there be any adverse customer impact or breaking change?

---

No correctness issues here, my suggestion is purely to be consistent with data-plane packages.